### ISSUE TITLE
Songs sometimes fail to load and thus do not play

### ISSUE THREAD

**Issue by PaulDance** [matthias4217/jukebox-ultra-nrv#49]
```
mai 14 23:12:21 hackademint python3.6[20735]: JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
mai 14 23:12:21 hackademint python3.6[20735]: ALSA lib pcm_dmix.c:1108:(snd_pcm_dmix_open) unable to open slave
mai 14 23:12:21 hackademint python3.6[20735]: couldn't open play stream: No such file or directory
mai 14 23:16:18 hackademint python3.6[20735]: ALSA lib pcm_dmix.c:1108:(snd_pcm_dmix_open) unable to open slave
mai 14 23:16:18 hackademint python3.6[20735]: Cannot connect to server socket err = No such file or directory
mai 14 23:16:18 hackademint python3.6[20735]: Cannot connect to server request channel
mai 14 23:16:18 hackademint python3.6[20735]: jack server is not running or cannot be started
mai 14 23:16:18 hackademint python3.6[20735]: JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
mai 14 23:16:18 hackademint python3.6[20735]: ALSA lib pcm_dmix.c:1108:(snd_pcm_dmix_open) unable to open slave
mai 14 23:16:18 hackademint python3.6[20735]: couldn't open play stream: No such file or directory
```

The jukebox seems to sometimes generate internal errors when trying to load a song that will then fail to play and the jukebox just goes to the next song in the current playlist. When trying again with the same song, it does not fail every time. The error does not seem to be consistent with resolution, length, volume, ...

**Comment by matthias4217** [comment]
For now there doesn't exist any flawless issue sorry (but it is planned to have one by th end of the week). Which commit are you on ?

I haven't had a similar log on my side. However the error you describe may be the one of #41. If it is the same, then it has been fixed in 1c4c12b.

**Comment by PaulDance** [comment]
> For now there doesn't exist any flawless issue sorry (but it is planned to have one by th end of the week). Which commit are you on ?
> 
> I haven't had a similar log on my side. However the error you describe may be the one of #41. If it is the same, then it has been fixed in [1c4c12b](https://github.com/matthias4217/jukebox-ultra-nrv/commit/1c4c12be6dea06e3a2528ccea9d94bac1ebf079b).

I had a version prior to the latest commits of today, but at the same time, the songs I selected were picked up from YouTube and not from the suggestions. I will try to update the repo on my side.

**Comment by matthias4217** [comment]
Okay, I've got the same problem when running it with systemd service. However it does not happen when running manually.
